projects/demo/src/modules/components/input-date-multi/input-date-multi.template.html

```diff
@@ -115,7 +115,7 @@
                 mobile calendar
             </a>
             , add imports of
-            <code>TuiMobileCalendarDialogModule</code>
+            <code>TuiMobileCalendarDialogComponent</code>
```

Comment:
That's wrong, you need to provide `TUI_MOBILE_CALENDAR`.

Reply:
fixed